platform/nephos/sonic-platform-modules-pegatron/debian/rules

```diff
@@ -35,10 +36,13 @@ clean:
 	dh_clean
 
 build:
-	#make modules -C $(KERNEL_SRC)/build M=$(MODULE_SRC)
+	# For fn-6254-dn-f ixgbe driver
+	git clone https://github.com/Peter5Lin/kernelDriver
+	git --git-dir=kernelDriver/.git/ --work-tree=kernelDriver am ../fn-6254-dn-f/$(MODULE_DIR)/pegatron_fn_6254_dn_f_ixgbe/0001-modify-Intel-ixgbe-driver-for-fn-6254-dn-f.patch
```

Comment:
we already have ixgbe driver, please use that. please submit patch to sonic-linux-kernel.

please have good description of the patch.

Reply:
Hi 

We need to use our own driver because our HW layout is different.
This patch is only for our platform.

Reply:
Hi

I will check optoe driver for SFP in few days

Reply:
it is hard to maintain another ixgbe driver.

Reply:
Hi

I understand, but the version I used is same as src/ixgbe.
To keep a stable version for us, I need to put it to another space instead of use 
`wget` -O ixgbe-$(IXGBE_DRIVER_VERSION).tar.gz "https://sonicstorage.blob.core.windows.net/packages/ixgbe-5.2.4.tar.gz?sv=2015-04-05&sr=b&sig=AaqJHHaPiJRp8R3HKobi0GNDgHAVnqijk6hpahwJ0Mg%3D&se=2154-10-05T22%3A19%3A29Z&sp=r"``

Or, I should use it directly?

Reply:
Hi
The modified was finished, please help to review
Thanks

Reply:
Hi 
The modified was done, please help to review
Thanks